mne/channels/montage.py

```diff
@@ -938,6 +1001,43 @@ def read_dig_montage(hsp=None, hpi=None, elp=None,
     return montage
 
 
+def read_dig_montage_brainvision(fname):
+    r"""Read subject-specific digitization montage from a brainvision file.
+
+    Parameters
+    ----------
+    bvct : path-like
+        BrainVision CapTrak coordinates file from which to read digitization
+        locations. This is typically in XML format. If str (filename), all
+        other arguments are ignored.
+
+    Returns
+    -------
+    montage : instance of DigMontage
+        The digitizer montage.
+
+    See Also
+    --------
+    DigMontage
+    Montage
+    read_montage
+    """
+    _check_fname(fname, overwrite='read', must_exist=True)
+    data = _parse_brainvision_dig_montage(fname)
+
+    # XXX: to change to the new naming in v.0.20 (all this block should go)
+    data.pop('point_names')
+    data['hpi_dev'] = data['hpi']
+    data['hpi'] = data.pop('elp')
```

Comment:
to me _parse_brainvision_dig_montage should not create the point_names, hpi_dev or hpi entries.

Reply:
I kept the same representation as much as possible and I was planning to change it at the end. All at once. Otherwise I'm never sure if I missed something